web/job_runs_controller_test.go

```diff
@@ -20,11 +22,40 @@ type JobRun struct {
 	ID string `json:"id"`
 }
 
+func BenchmarkJobRunsController_Index(b *testing.B) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+
+	b.ResetTimer()
+	for n := 0; n < b.N; n++ {
+		_, resp := testJobRunsControllerIndex(b, app)
+		assert.Equal(b, 200, resp.StatusCode, "Response should be successful")
+	}
+}
+
 func TestJobRunsController_Index(t *testing.T) {
 	t.Parallel()
+
 	app, cleanup := cltest.NewApplication()
 	defer cleanup()
 
+	j, resp := testJobRunsControllerIndex(t, app)
+
+	assert.Equal(t, 200, resp.StatusCode, "Response should be successful")
+	var respJSON JobRunsJSON
+	assert.Nil(t, json.Unmarshal(cltest.ParseResponseBody(resp), &respJSON))
+
+	jr, err := app.Store.JobRunsFor(j.ID)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	assert.Equal(t, 2, len(respJSON.Runs), "expected no runs to be created")
+	assert.Equal(t, jr[0].ID, respJSON.Runs[0].ID, "expected runs ordered by created at(descending)")
+	assert.Equal(t, jr[1].ID, respJSON.Runs[1].ID, "expected runs ordered by created at(descending)")
+}
+
+func testJobRunsControllerIndex(t assert.TestingT, app *cltest.TestApplication) (*models.JobSpec, *http.Response) {
```

Comment:
I'd move the initial saving of records out of the shared logic. Otherwise, we're actually benchmarking how long it takes to save 2 records and hit the index API. Really the goal is to get an accurate representation of how fast the index API is.